core/domain/feedback_domain.py

```diff
@@ -97,25 +161,54 @@ def to_dict(self):
 class FeedbackAnalytics(object):
     """Domain object representing feedback analytics
     for a specific exploration.
+    
+    Attributes:
+        exploration_id: str. The associated exploration's ID.
+        num_open_threads: int. The number of open threads under 
+            a given exploration.
+        num_total_threads: int. The number of total threads under 
+            a given exploration (regardless of status).
     """
 
     def __init__(self, exploration_id, num_open_threads, num_total_threads):
+        """Initializes a FeedbackAnalytics object.
+        """
```

Comment:
This was handled by attributes and left alone per feedback from @seanlip on Dec 24, 2016. So I'll say this is done.